services/web/server/src/simcore_service_webserver/security/_authz_access_roles.py

```diff
@@ -32,6 +32,8 @@ class PermissionDict(TypedDict, total=False):
     UserRole.ANONYMOUS: PermissionDict(can=[]),
     UserRole.GUEST: PermissionDict(
         can=[
+            "folder.read",
+            "folder.update",
```

Comment:
Guest users have no access to the Dashboard, I wouldn't let them read or update folders.